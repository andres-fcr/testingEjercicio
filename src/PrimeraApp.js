import React from 'react'

const PrimeraApp = ({saludo, subtitulo}) => {
  return (
    <div>
        <h1>{saludo}</h1>
        <p>{subtitulo}</p>
    </div>
  )
}

export default PrimeraApp