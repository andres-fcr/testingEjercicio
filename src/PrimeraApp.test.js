import { shallow } from "enzyme"
import PrimeraApp from "./PrimeraApp"

describe('Prueba <PrimeraApp />', () => {
    test('Validar render de <PrimeraApp/>', () => {
        const saludoA = 'Hola mundo'
        const subitutloA = 'Esto es una prueba'

        const wrapper = shallow(<PrimeraApp saludo={saludoA} subtitulo={subitutloA} />)
        expect(wrapper).toMatchSnapshot()
    })
})