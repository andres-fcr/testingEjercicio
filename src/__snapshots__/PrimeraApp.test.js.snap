// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba <PrimeraApp /> Validar render de <PrimeraApp/> 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
  <p>
    Esto es una prueba
  </p>
</div>
`;
